api/app/serializers/spree/api/v2/platform/store_serializer.rb

```diff
@@ -6,7 +6,7 @@ class StoreSerializer < BaseSerializer
           include ::Spree::Api::V2::ResourceSerializerConcern
 
           has_many :menus
-          has_one :default_country, serializer: :country, record_type: :country, id_method_name: :default_country_id
+          has_one :default_country, serializer: :country, record_type: :country, id_method_name: :default_country_iso
```

Comment:
@v10110 this shouldn't change, we're not changing the association between Store and Country